python/tank/bootstrap/constants.py

```diff
@@ -19,7 +19,7 @@
 # if major changes happen to the way cloud based configs are handled
 # by the system, for example requiring any existing deployed cloud
 # configs to be re-deployed, this version number should be incremented.
-BOOTSTRAP_LOGIC_GENERATION = 5
+BOOTSTRAP_LOGIC_GENERATION = 6
```

Comment:
We need to bump the logic generation so the files get rewritten the next time we bootstrap into a project so the issue is fixed.